src/index.html

```diff
@@ -10,8 +10,211 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>HTML Form</h1>
+  <body class="body">
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend class="legend">Personal information</legend>
+
+        <div class="container">
+          <label class="label" for="surname">
+            Surname:
+            <input
+              type="text"
+              name="surname"
+              autocomplete="off"
+              id="surname"
+              required
+            >
+          </label>
```

Comment:
no need for 'for' attribute for label if you wrap input into label

```suggestion
          <label class="label">
            Surname:
            <input
              type="text"
              name="surname"
              autocomplete="off"
              required
            >
          </label>
```